pkg/status/status.go

```diff
@@ -85,10 +87,45 @@ func GetStatus() (map[string]interface{}, error) {
 	}
 
 	stats["systemProbeStats"] = getSystemProbeStats()
+	stats["apmStats"] = GetAPMStatus()
 
 	return stats, nil
 }
 
+// GetAPMStatus returns a set of key/value pairs summarizing the status of the trace-agent.
+// A key "outcome" is always returned and it may be either "error" or "success".
+func GetAPMStatus() map[string]interface{} {
+	port := 8126
+	if p, ok := os.LookupEnv("DD_APM_RECEIVER_PORT"); ok {
+		if v, err := strconv.Atoi(p); err == nil {
+			port = v
+		}
+	}
+	if config.Datadog.IsSet("apm_config.receiver_port") {
+		port = config.Datadog.GetInt("apm_config.receiver_port")
+	}
+	url := fmt.Sprintf("http://localhost:%d/debug/vars", port)
+	resp, err := (&http.Client{Timeout: 3 * time.Second}).Get(url)
```

Comment:
I 3s too much ? This would block the status page for a while. And I don't see why the trace agent would take so long. 1s maybe ? Also could we move this default to the config, just in case ?

Reply:
You're right. It _is_ too much. It's what we had in the trace-agent so I technically just copied it. I'm changing to 1s, not sure if this warrants a config.